src/Display/PointingModal/PointingModal.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+
+import PointingModal, { Props } from './PointingModal';
+import ProfilePicture from '../../General/ProfilePicture';
+import Divider from '../../General/Divider';
+import { BaseContainer } from '../../Layout/GlintsContainer/GlintsContainer';
+
+export default {
+  title: 'Display/PointingModal',
+  component: PointingModal,
+  decorators: [Story => <BaseContainer>{Story()}</BaseContainer>],
+} as Meta;
+
+const Template: Story<Props> = ({ ...args }) => (
+  <PointingModal {...args}>
+    <PointingModal.Header>
+      <ProfilePicture editable>
+        <img
+          src="https://images.unsplash.com/photo-1512979797260-1a645592b48f?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=2fc2ca84d8c407bd48ab15ea27c87eaf&auto=format&fit=crop&w=750&q=80"
+          alt="profile"
+        />
+      </ProfilePicture>
+      <div className="aries-modal-profile">
+        <span>Hello,</span>
+        <a href=".">Fredy Yanto</a>
+      </div>
+    </PointingModal.Header>
+    <Divider theme="grey" />
+    <PointingModal.Body>
```

Comment:
For `PointingModal.Body` and `PointingModal.Header`, I think we can add `displayName` to display correct usage in code sample. Can refer to `Accordion.Panel.displayName = 'Accordion.Panel'`

![image](https://user-images.githubusercontent.com/13249748/104431838-5223e580-55c3-11eb-9821-c312dbeb0af5.png)

Reply:
Ah yes, good idea!